public/app/plugins/datasource/prometheus/querycache/QueryCache.test.ts

```diff
@@ -5,10 +5,21 @@ import { DataFrame, DataQueryRequest, DateTime, dateTime, TimeRange } from '@gra
 import { QueryEditorMode } from '../querybuilder/shared/types';
 import { PromQuery } from '../types';
 
-import { getTargSig, QueryCache } from './QueryCache';
+import { DatasourceProfileData, QueryCache } from './QueryCache';
 import { IncrementalStorageDataFrameScenarios } from './QueryCacheTestData';
 
-const mockRequest = (request?: Partial<DataQueryRequest<PromQuery>>): DataQueryRequest<PromQuery> => {
+// Will not interpolate vars!
+const interpolateStringTest = (query: PromQuery) => {
+  return query.expr;
+};
+
+const getPrometheusTargetSignature = (request: DataQueryRequest<PromQuery>, targ: PromQuery) => {
+  return `${interpolateStringTest(targ)}|${targ.interval ?? request.interval}|${JSON.stringify(
```

Comment:
Not sure why this function is called interpolate if it doesn't interpolate

Reply:
See the above, it's just named this because in the actual implementation this function needs to interpolate any variables